docs/migrating.md

```diff
@@ -6,10 +6,10 @@ While Warp maintains as much backward-compatibility as possible from Fabric,
 some breaking changes were necessary or justified.
 
 ### How to migrate
-1. Make sure to install warp to your project. Follow the instructions on [getting started](https://warp-ds.github.io/tech-docs/getting-started/)
-2. Remove Fabric as a dependency from your project
-3. Remove any mentions of Fabric, e.g. `fabric.css`
-4. Change the prefixes `f-` to `w-` for your component names. The icons should still use the `f-` prefix. Use the [migration plugin](https://warp-ds.github.io/tech-docs/getting-started/#migration-plugin) to help with this (see further down this page for further instructions on using the plugin)
+1. Remove Fabric as a dependency from your project.
+2. Remove any mentions of Fabric, e.g. `fabric.css`.
+3. Change the prefixes `f-` to `w-` for your component names. The icons should still use the `f-` prefix. Use the [migration plugin](https://github.com/warp-ds/preset-migrate) to help with this (see further down this page for further instructions on using the plugin).
```

Comment:
Maybe remove the second `further`?
_see further down this page for instructions_

Reply:
[a454263](https://github.com/warp-ds/tech-docs/pull/47/commits/a454263ecefbd6f2c383a3d40e381e3007487067)